extensions/wrapper/wrapper/src/main/java/de/sovity/edc/ext/wrapper/api/ui/model/CriterionDto.java

```diff
@@ -0,0 +1,40 @@
+/*
+ *  Copyright (c) 2022 sovity GmbH
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       sovity GmbH - initial API and implementation
+ *
+ */
+
+package de.sovity.edc.ext.wrapper.api.ui.model;
+
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.NoArgsConstructor;
+import lombok.AllArgsConstructor;
+
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+public class CriterionDto {
+    private String operandLeft;
+    private String operator;
```

Comment:
We might want to replace the operator with the core edc operator enum.

While we shouldn't leak core edc models, I think using the operator enum to limit the possible values might be worth it, as deserialization will definitely fail if it is not a value available in the enum.

There are two possibilities: Leak the enum or create a copy of the enum with `@Schema` and a test that tests the equality of the enums.